nengo_bones/scripts/generate_bones.py

```diff
@@ -131,128 +90,87 @@ def ci_scripts(ctx):
     config = ctx.obj["config"]
     for params in config["ci_scripts"]:
         script_name = params.pop("template")
-        template = ctx.obj["env"].get_template(script_name + ".sh.template")
-        output_name = params.pop("output_name", script_name)
-        path = os.path.join(ctx.obj["output_dir"], output_name + ".sh")
-
-        with open(path, "w") as f:
-            f.write(template.render(
-                pkg_name=config["pkg_name"], repo_name=config["repo_name"],
-                version=nengo_bones.__version__, **params))
-
-        # Mark CI script as executable
-        st = os.stat(path)
-        os.chmod(path, st.st_mode | stat.S_IEXEC)
+        output_file = params.pop("output_name", script_name)
+        BonesTemplate(script_name + ".sh").render_to_file(
+            ctx.obj["output_dir"],
+            output_name=output_file + ".sh",
+            version=__version__,
```

Comment:
Note: remove this after the commit removing version numbers from templates

Reply:
Not sure where you think it should be removed ... currently it gets removed in the same commit all the other version stuff is done ?